subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -1087,6 +1094,12 @@ public void preventFromFurtherMutation() {
         }
     }
 
+    @Override
+    public boolean isIncubating() {
```

Comment:
Currently, this only checks for this specific attribute, which all 3 new variants possess.  Should I expand this to test for all the new `TestSuiteName`, `TestSuiteType`, etc., types and values, or leave it as is?